pwa/src/templates/templateParts/header/HeaderTemplate.tsx

```diff
@@ -116,6 +129,13 @@ export const HeaderTemplate: React.FC<HeaderTemplateProps> = ({ layoutClassName
             navigate("/documentation/standards");
           },
         },
+        {
+          label: t("Contact"),
+          current: pathname === "/documentation/contact",
+          handleClick: () => {
+            navigate("/documentation/contact");
```

Comment:
Right now this navigate links to a non existing page. I think this meant to be a "#"